sar/README.md

```diff
@@ -119,20 +119,22 @@ only additional requirements and mappings to fulfill the CARD4L requirements are
+| card4l:integrated_sidelobe_ratio       | ✓ 1.6.9     | ✗           | `IntegratedSideLobeRatio`                                    | number                                                  | Integrated sidelobe ratio (ISLR) in decibel. |
 | card4l:noise_removal_applied           | (✓)         | ✓ 3.3       | `NoiseRemovalApplied`                                        | boolean                                                 | **REQUIRED for *Prod*.** Specifies whether noise removal has been applied (`true`) or not (`false`). If set to `true`, a [link with relation type](#stac-item-links) `noise-removal` is **required**, too. |
-| card4l:mean_faraday_rotation_angle     | ✓ 1.6.11    | ✗           | `MeanFaradayRotationAngle`                                   | number                                                  | Convert to degree, if required.                              |
+| card4l:mean_faraday_rotation_angle     | ✓ 1.6.11    | ✗           | `MeanFaradayRotationAngle`                                   | number                                                  | In degrees. |
 | card4l:ionosphere_indicator            | ✓ 1.6.12    | ✗           | `IonosphereIndicator`                                        | boolean                                                 | Flag indicating whether the imagery is “significantly impacted” by the ionosphere (`false` - no, `true` – yes). |
 | card4l:speckle_filtering               | ✗           | ✓ 1.7.4     | `Filtering`, `FilterApplied`                                 | [Speckle Filter Object](#speckle-filter-object) \| null | **REQUIRED.** Set to `null` if `FilterApplied` would be set to `false`. Otherwise, provide a [Speckle Filter Object](#speckle-filter-object). |
 | card4l:pixel_coordinate_convention     | ✗           | ✓ 1.7.8     | `PixelCoordinateConvention`                                  | string                                                  | **REQUIRED.** One of `center` (pixel center), `upper-left` (pixel ULC) or `lower-left` (pixel LLC) |
 | card4l:measurement_type                | ✗           | ✓ 3.1       | `BackscatterMeasurement` (NRB)                               | string                                                  | **REQUIRED.** Must be set to `gamma0`.                       |
-| card4l:measurement_convention          | ✗           | ✓ 3.1       | `BackscatterConvention` (NRB)                                | string                                                  | **REQUIRED.** Must be set to `amplitude`, `power` (both NRB + POL) or `angle` (POL only). |
+| card4l:measurement_convention          | ✗           | ✓ 3.1       | `BackscatterConvention` (NRB)                                | string                                                  | **REQUIRED.** Must be set to `linear amplitude`, `linear power` (both NRB + POL) or `angle` (POL only). |
```

Comment:
This would also need to be changed in the JSON Schema. Is it worth breaking existing implementations for (just?) adding `linear` in the values? Would the proposal below also work? So keeping the values in the JSON files as is, but clarifying here in the docs that they are meant to be linear?

```suggestion
| card4l:measurement_convention          | ✗           | ✓ 3.1       | `BackscatterConvention` (NRB)                                | string                                                  | **REQUIRED.** Must be set to linear `amplitude`, linear `power` (both NRB + POL) or `angle` (POL only). |
```

Reply:
Thanks, I also noticed that I missed adjusting the schema! 
The [XLSX v5.5.5](https://ceos.org/ard/files/PFS/NRB/v5.5/CARD4L_METADATA-spec_NRB-v5.5.5.xlsx) now clearly lists `[linear amplitude, linear power]` as standard parameters (see also the 'v5.5 Changes' tab), so I'd suggest changing this properly now.

Reply:
Yeah, I see the point of aligning. On the other hand, we have a couple of implementations that use the existing values and changing them between versions would make implementations that read that data a bit more annoying. The mapping is still 1:1 and I'm not sure we need to adopt the CEOS XML recommendations 1:1, we also don't do that in other places. Not sure...

Reply:
With growing maturity of the CARD4L specifications, there are going to be fewer changes in the future between versions. What's the point in having this extension if we don't align with the specification 1:1 in the process? I think it will be worthwhile for everyone in the long run. If this extension deviates from the specification documents too much in the future it will result in a confusing mess in my opinion (for users and especially people who implement this into an NRB data product, for example).